mlrun/projects/project.py

```diff
@@ -2669,17 +2669,20 @@ def set_secrets(
         file_path: str = None,
         provider: typing.Union[str, mlrun.common.schemas.SecretProviderName] = None,
     ):
-        """set project secrets from dict or secrets env file
+        """
+        Set project secrets from dict or secrets env file
         when using a secrets file it should have lines in the form KEY=VALUE, comment line start with "#"
         V3IO paths/credentials and MLrun service API address are dropped from the secrets
 
-        example secrets file::
+        example secrets file:
+
+        .. code-block:: text
```

Comment:
How about this?
```suggestion
        .. code-block:: shell
```

Reply:
Done :)
Indeed, it looks better:
![image](https://github.com/mlrun/mlrun/assets/36337649/c1687576-ae06-450c-9797-cef0cc188758)